docs/components/authentication/sign-in.mdx

```diff
@@ -92,61 +92,43 @@ Below is basic implementation of the `<SignIn />` component. You can use this as
 
     <InjectKeys>
 
-    <CodeBlockTabs type="npm-script" options={['NPM Module', 'window.Clerk']}>
-      ```typescript
+    <CodeBlockTabs type="npm-script" options={['NPM module', '<script>']}>
```

Comment:
These updates are good, but instead of having a JavaScript tab here we should probably have a callout in the intro for this page letting users know this is the React component, and they should go to `/docs/references/javascript/clerk/sign-in` to use SignUp in JavaScript.

Having the info in both places makes it easier for discrepancies to accrue over time, and makes navigating the site via docs search or google more confusing.